### DISCUSSION TITLE
Regarding Reapply for Github student developer pack

### DISCUSSION THREAD

**Discussion by ChinmayKaitade** [community/community#132523] [GitHub Education]
### Hi everyone,

Hello guys, I applied for the GitHub Developer Pack but I forgot to enter the billing address, so my application was rejected. It mentioned that I should reapply by entering the billing address, but the reapply option is not showing. How can I reapply? Please let me know.

**Comment by thiunuwan** [comment]
If the reapply option is not showing, here are some steps you can try to resolve the issue:

- Clear Browser Cache and Cookies: Sometimes clearing the cache and cookies can resolve issues with website functionality.

- Use a Different Browser or Device: Try accessing the GitHub Education page from a different browser or device.

- Contact GitHub Support: If the reapply option is still not available, contact GitHub support
